substrate/frame/core-fellowship/src/lib.rs

```diff
@@ -163,6 +163,13 @@ impl<
 	}
 }
 
+pub struct ConvertU16ToU32<Inner>(PhantomData<Inner>);
```

Comment:
Fine for now. After the pr got merged, could you do a commit to the upstream crate that has the `Get` trait to introduce some sort of `GetInto` converter type:

```
impl<T: Get<I>, I: Into<R>, R> Get<R> GetInto<I> {
       fn get() -> R {
              T::get().into()
       }
}

Reply:
Underway https://github.com/paritytech/parity-common/pull/909